x-pack/solutions/security/plugins/security_solution/public/onboarding/components/onboarding_body/cards/search_ai_lake/llm/index.ts

```diff
@@ -10,19 +10,19 @@ import { AssistantIcon } from '@kbn/ai-assistant-icon';
 import type { OnboardingCardConfig } from '../../../../../types';
 import { OnboardingCardId } from '../../../../../constants';
 import { AI_CONNECTOR_CARD_TITLE } from './translations';
-import type { AIConnectorCardMetadata } from './types';
 import { SECURITY_FEATURE_ID } from '../../../../../../../common/constants';
 import { checkAssistantCardComplete } from '../../common/connectors/assistant_check_complete';
+import type { AssistantCardMetadata } from '../../common/connectors/types';
 
-export const llmConnectorCardConfig: OnboardingCardConfig<AIConnectorCardMetadata> = {
+export const llmConnectorCardConfig: OnboardingCardConfig<AssistantCardMetadata> = {
   id: OnboardingCardId.searchAiLakeLLM,
   title: AI_CONNECTOR_CARD_TITLE,
   icon: AssistantIcon,
   Component: React.lazy(
     () =>
       import(
         /* webpackChunkName: "onboarding_search_ai_lake_llm_connector_card" */
-        './llm_connector_card'
+        '../../common/connectors/assistant_card'
```

Comment:
Couldn't we use the same card? I mean literally, instead of moving the card implementation to /common and loading it from two different places. Is there any differences between them?

Reply:
Just the title is different,
AI4SOC: Configure AI Provider
Complete / essentials: Configure AI Assistant

Reply:
🤦 Let's use "Configure AI Provider" everywhere